src/tui/mode/mini/match.zig

```diff
@@ -0,0 +1,28 @@
+const std = @import("std");
+const cbor = @import("cbor");
+const command = @import("command");
+const tp = @import("thespian");
+const log = @import("log");
+
+const tui = @import("../../tui.zig");
+
+pub const Type = @import("get_char.zig").Create(@This());
+pub const create = Type.create;
+
+pub fn name(self: *Type) []const u8 {
+    var suffix: []const u8 = "";
+    if ((self.ctx.args.match(.{tp.extract(&suffix)}) catch false)) {
+        return suffix;
+    }
+    return "󰅪 match";
+}
+
+pub fn process_egc(self: *Type, egc: []const u8) command.Result {
+    var action: []const u8 = "";
+    if ((self.ctx.args.match(.{tp.extract(&action)}) catch false)) {
+        try command.executeName(action, command.fmt(.{egc}));
+    } else {
+        try command.executeName("match_brackets", .{});
```

Comment:
Should `egc` be passed to `match_brackets` here? If not, the mini mode has no effect at all right?

Reply:
Totally, thank you, removed, also improved mode detection for bracket match...

Reply:
Yeah, I'm not sure detecting modes via the mode name is a good idea. Anybody can call their mode "SEL" which will then lead to special behavior, which is really surprising and likely a POLA violation.

It's probably better to avoid detecting modes at all and instead request the required functionality explicitly instead. That way it's always possible to reuse the feature from other contexts or modes.

I will merge this now anyway, but you might want to think about a different approach later.